# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1484692123 18000
#      Tue Jan 17 17:28:43 2017 -0500
# Node ID 9ca0829b160e4bb48412ba1afa1ff5e2c6c710e7
# Parent  aedc662896a3f2dbb382cd5b793a6372c518747c
fix editing error in previous change to oct-fftw.cc (bug #50070)

* oct-fftw.cc: Move #if inside of namespace octave.

diff --git a/liboctave/numeric/oct-fftw.cc b/liboctave/numeric/oct-fftw.cc
--- a/liboctave/numeric/oct-fftw.cc
+++ b/liboctave/numeric/oct-fftw.cc
@@ -37,20 +37,20 @@ along with Octave; see the file COPYING.
 #include "quit.h"
 #include "oct-locbuf.h"
 #include "singleton-cleanup.h"
 
 #if defined (HAVE_FFTW3_THREADS) || defined (HAVE_FFTW3F_THREADS)
 #  include "nproc-wrapper.h"
 #endif
 
+namespace octave
+{
 #if defined (HAVE_FFTW)
 
-namespace octave
-{
   fftw_planner *fftw_planner::instance = 0;
 
   // Helper class to create and cache FFTW plans for both 1D and
   // 2D.  This implementation defaults to using FFTW_ESTIMATE to create
   // the plans, which in theory is suboptimal, but provides quite
   // reasonable performance in practice.
 
   // Also note that if FFTW_ESTIMATE is not used then the planner in FFTW3
